libraries/common/helpers/config/mock.js

```diff
@@ -1,33 +1,27 @@
 /**
- * @typedef {Object} ThemeColors
- * @property background
- * @property light
- * @property dark
- * @property accent
- * @property placeholder
- * @property primary
- * @property darkGray
- * @property shade3
- * @property shade4
- * @property shade5
- * @property shade6
- * @property shade7
- * @property shade8
- * @property shade9
- * @property shade10
- * @property shade11
- * @property shade12
- * @property success
- * @property warning
- * @property error
+ * @mixin ThemeColors
  */
 const colors = {
   background: '#f8f8f8',
   light: '#fff',
+  lightTransparent: 'rgba(255, 255, 255, 0.82)',
   dark: '#000',
+  lightDark: '#323232',
   accent: '#5ccee3',
   placeholder: '#f2f2f2',
+  lightOverlay: '#fafafa',
+  overlay: '#f7f7f7',
+  darkOverlay: '#f0f0f0',
   primary: '#fa5400',
+  primaryContrast: '#fff',
+  accentContrast: '#fff',
+  cta: '#fa5400',
+  ctaContrast: '#fff',
+  focus: '#fa5400',
+  gray: '#747474',
+  lightGray: '#9a9a9a',
+  lightGrayTransparent: 'rgba(0,0,0, 0.4)',
```

Comment:
This is more `darkTransparent`, isn't it?

Reply:
By color codes yes. But visually differs
![image](https://user-images.githubusercontent.com/18292310/75446333-7d7bf280-5967-11ea-9e76-5c0d5ded5447.png)